swan-lake/development-tutorials/source-code-dependencies/manage-dependencies.md

```diff
@@ -64,35 +64,44 @@ The following is another example that shows the usage of multiple modules from d
 ```ballerina
 // Imports the default module from the `ballerina/log` package.
 import ballerina/log;
-// Imports the default module from the `googleapis.gmail` package
+// Imports the default module from the `salesforce` package
 // with an import prefix.
-import ballerinax/googleapis.gmail as gmail;
-// Imports the only non-default module from the `googleapis.gmail` package
+import ballerinax/salesforce as salesforce;
```

Comment:
We usually use an alias when we want to change the default prefix. So let's change to something like `sf`